twenty-first/src/util_types/merkle_tree.rs

```diff
@@ -950,7 +1020,7 @@ pub mod merkle_tree_test {
         //
         //  0      2   <-- opened_leaf_indices
 
-        let node_indices = [3, 8, 9, 10, 11];
+        let node_indices = [3 as MerkleTreeNodeIndex, 8, 9, 10, 11];
```

Comment:
Type inference can figure this out by itself.

```suggestion
        let node_indices = [3, 8, 9, 10, 11];
```